internal/server/handlers.go

```diff
@@ -176,6 +178,15 @@ func (a *API) Login(c *gin.Context, r *api.LoginRequest) (*api.LoginResponse, er
 
 	setAuthCookie(c, bearer, key.ExpiresAt)
 
+	if name == "" {
+		user, err := access.GetIdentity(c, key.IssuedFor)
+		if err == nil {
+			name = user.Name
+		}
+	}
+	a.t.User(key.IssuedFor.String(), name)
+	a.t.OrgMembership(key.OrganizationID.String(), key.IssuedFor.String())
```

Comment:
Might make sense to combine these into one event where they are tracking similar things

Reply:
Wasn't exactly sure how this shows up in segment, but I was thinking they're separate things.